### ISSUE TITLE
MD Toolip Safari - first hover - flies in from top left hand corner

### ISSUE THREAD

**Issue by simonmorley** [angular/material#8818]
The tooltip appears to fly-in from the top LHS on first hover. Thereafter, it appears in the correct position - i.e. at the bottom.

Only happens in Safari on OSX. Chrome and FF are OK.

```
<span>
  Clock strikes upon the hour
  <md-tooltip>
    The sun begins to fade
  </md-tooltip>
</span>
```

Tested on Version 9.1 (11601.5.17.1) using Angular Material 1.1.0-rc.5 & rc4, 1.0.9.

![naughty-safari](https://cloud.githubusercontent.com/assets/309336/16232772/c5520dca-37c3-11e6-8719-7ef0bb269102.gif)

**Comment by lolptdr** [comment]
As a temporary fix, try changing the <md-tooltip> element's scss to "position:fixed". I've encountered this issue on Chrome 51 since I was using a unique wrapping of other element tags that probably overrode the positioning. You can use a vendor prefix to target safari also.

**Comment by codeandqa** [comment]
I am still seeing this issue in IE and firefox.

**Comment by Splaktar** [comment]
@codeandqa what version of AngularJS Material are you seeing this with? It should have been fixed in 1.1.0. Please open a new issue with a CodePen demo if you can reproduce it with AngularJS Material 1.1.7.
